.env.example

```
@@ -288,8 +288,9 @@ PYTHON_BACKEND_GRPC_ADDR=127.0.0.1:9092
 # -----------------------------------------------------------------------------
 PROMETHEUS_PORT=9090
 GRAFANA_PORT=3000
-GRAFANA_ADMIN_USER=admin
-GRAFANA_ADMIN_PASSWORD=admin
+# Generate strong credentials: python3 -c "import secrets; print(secrets.token_urlsafe(24))"
+GRAFANA_ADMIN_USER=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GRAFANA_ADMIN_USER key should go before the GRAFANA_PORT key

(UnorderedKey)